acceleration/README.md

```diff
@@ -0,0 +1,166 @@
+# Sample
+
+This sample showcases the way to use spiced to simplify the process of building a CQRS pattern for your application.
```

Comment:
Don't call it `spiced`, let's use the same wording we use in the Docs/Readme

Reply:
Please resolve / address in next iteration.  Closing comment to merge.